src/services/API/otp.js

```diff
@@ -0,0 +1,34 @@
+import {OTPClient} from "@/services/API/rootClient.js"
+import { sendOTPEndpoint, verifyOTPEndpoint } from "@/services/API/endpoints.js";
```

Comment:
pre commit is not working on js files?
on line 1 there's no space between the brackets and the `OTPClient` string, but on line 2 there is space present.
Can you please check if the pre commit is working on js files or not. Beacuse I think it's working fine on .vue files